### PR TITLE
Added loading indication

### PR DESCRIPTION
**Pull request by alien1976**
[Task](https://coherent-labs.atlassian.net/browse/COH-17559)
Added spinner when the buttons are pressed and added a ready message for the buttons that when are pressed generate files.

Cohtml PR - >https://github.com/CoherentLabs/cohtml/pull/2265

### CODE REVIEW

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts [hunk 1]
```diff
@@ -224,12 +224,13 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
...
-  private fetchCacheStates() {
-    const cacheStats = this.cohtmlDebugModel?.getRenoirCahcesState();
-    cacheStats?.then((stats) => {
-      // update the HTML elements displaying the currenet capacity count and bytes
-      this.updateReniorCachesUI(stats);
-    });
+  private async fetchCacheStates() {
+    if (!this.cohtmlDebugModel) return;
+
+    const stats = await this.cohtmlDebugModel?.getRenoirCahcesState();
```
Comment (venetak) [inline front_end/entrypoints/inspector_main/CohtmlPanel.ts]: We check if it exists literally on the previous line.

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts [hunk 2]
```diff
@@ -283,13 +284,19 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
...
       const entryButton = UI.UIUtils.createTextButton(i18nString(UIStrings.updateCacheStr), void 0, 'cohtml-button margin-top-0');
       UI.ARIAUtils.markAsLink(entryButton);
+      entryButton.disabled = !entryInput.value;
+      //@ts-ignore
+      entryInput.addEventListener('input', (event) => entryButton.disabled = !event.target.value)
       buttonWrapper.appendChild(entryButton);
 
-      entryButton.onclick = () => {
+      const loadingIndicator = buttonWrapper.createChild('div');
+      loadingIndicator.classList.add('icon');
```
Comment (venetak) [inline front_end/entrypoints/inspector_main/CohtmlPanel.ts]:
```suggestion
      const loadingIndicator = buttonWrapper.createChild('div', 'icon');
```

**Review by venetak — APPROVED** [review]

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts (hunk 2 above)
Comment (alien1976) [reply front_end/entrypoints/inspector_main/CohtmlPanel.ts]: Fixed and refactored usage everywhere of classlist.add

File: front_end/entrypoints/inspector_main/CohtmlPanel.ts (hunk 1 above)
Comment (alien1976) [reply front_end/entrypoints/inspector_main/CohtmlPanel.ts]: Yep... Didn't realize that .. :D Fixed

**Review by Tony-CL — APPROVED** [review]

**Pull request merged**